packages/docs/components/atoms/Cursor/index.md

```diff
@@ -12,3 +12,26 @@ Use the cursor component to easily add a custom cursor to your project.
 - [Examples](./examples.html)
 - [JS API](./js-api.html)
 - [Twig API](./twig-api.html)
+
+## Usage
+
+Once the [package installed](/guide/installation/), simply include the template in your project:
+
+```js {2,8}
+import { Base, createApp } from '@studiometa/js-toolkit';
+import { Cursor } from '@studiometa/ui';
+
+class App extends Base {
+  static config = {
+    name: 'Base',
+    components: {
+      Cursor,
+    }
+  };
+}
+
+export default createApp(App);
+```
+```twig
+{% include '@ui/atoms/Cursor/Cursor.twig' %}
```

Comment:
**question** Should add the `only` attribute to the exemple ? To follow best-practice

Reply:
Yes!